handbook/engineering/web/goals.md

```diff
@@ -39,9 +54,9 @@ The individual tasks and progress of the current iteration can be found as GitHu
 1. ✅ Existing sourcegraph extensions are more discoverable ([RFC 209](https://docs.google.com/document/d/1I5BMEGp3QuB81AjSzLCQwq_XJV1sXevlU0lpB4O1pj8/edit#))
 1. ✅ The Sourcegraph browser extension is more discoverable and easy to congifure ([RFC 221](https://docs.google.com/document/d/19f4xleYBU1zZZdqMmXlLmFxeR-fwEpOwTOgViOFOnyo/edit))
 1. ✅ Build new and improved Sourcegraph extensions to showcase the value and opportunity of extensions ([RFC 246](https://docs.google.com/document/d/1HngEeLNAe7_QzVJr6UPi0Si4ZALqTzb7uonOxUiJP6g/edit))
-1. 🔄 Improve the Sourcegraph extensions (internal) development experience ([RFC 155](https://docs.google.com/document/d/1ikrUNVe3YVbR-JpegxhjrFdmRkTGzTLcOMkKHnOyjuE/edit)) and (external) documentation
-1. Code insights TBD
+1. ✅ Improve the Sourcegraph extensions (internal) development experience ([RFC 155](https://docs.google.com/document/d/1ikrUNVe3YVbR-JpegxhjrFdmRkTGzTLcOMkKHnOyjuE/edit)) and (external) documentation
+1. 🔄 Code insights migration prototype and directory decoration
+1. 🔄 Safari browser extension
```

Comment:
Is Gerrit support in our short-term plans?

Reply:
Yep, and glad that you're tracking! I haven't update the short term goals yet beyond these present-time updates ("short term goals on their way!" in the [PR description](https://github.com/sourcegraph/about/pull/1950#issue-515831650)). I wanted to separate the goals from the short term roadmap for the sake of "let's agree we have these goals" before "let's agree this work gets us to these goals." Short term goals PR coming next now.